pkg/action/executor/rescheduler.go

```diff
@@ -149,13 +149,19 @@ func (r *ReScheduler) preActionCheck(pod *api.Pod, node *api.Node) error {
 	}
 
 	for _, cond := range conditions {
-		if cond.Status != api.ConditionTrue {
-			glog.Warningf("Move action: pod[%v]'s new host(%v) in bad condition: %v", fullName, node.Name, cond.Type)
+		if cond.Type == api.NodeReady {
+			// If the destination node is NOT in a Ready state, return an error to fail the action
+			if cond.Status != api.ConditionTrue {
+				return fmt.Errorf("Move action: pod[%v]'s new host (%v) is in bad condition: %v",
```

Comment:
We should just say that the new host is NotReady

Reply:
Updated the error message.